components/ui/NotificationCenter.tsx

```diff
@@ -109,7 +109,7 @@ const NotificationCenter = () => {
   const { notifications, removeNotification } = useNotifications();
 
   return (
-    <div className="pointer-events-none fixed top-0 z-[100] h-full w-full">
+    <div className="pointer-events-none fixed top-0 z-[20] h-full w-full">
```

Comment:
Have you verified that this doesn't cause the notifications to drop behind anything? I'm thinking it might fall behind the modal overlay